drafts/web-editor-monaco-with-language-server-support.md

```diff
@@ -0,0 +1,971 @@
+---
+title: Create a custom web editor using Monaco and Language Server Protocol (LSP)
+domain: software-engineering-corner.hashnode.dev
+tags: Web Editor, Monaco, Monaco Language Client, Language Server, LSP
```

Comment:
Regarding the tags, in our readme its recommented to `Use tags which are listed here https://github.com/Hashnode/support/blob/main/misc/tags.json (use the slug) or ensure they exist via the Hashnode search (select "tags"). If you do it wrong, Hashnode may fail to import the article.`